executor/cook/config.py

```diff
@@ -48,21 +49,25 @@ def __init__(self,
         self.sandbox_directory = sandbox_directory
         self.shutdown_grace_period_ms = ExecutorConfig.parse_time_ms(shutdown_grace_period)
 
+    def stdout_file(self):
+        return os.path.join(self.sandbox_directory, 'stdout')
+
 
 def initialize_config(environment):
     """Initializes the config using the environment.
     Populates the default values for missing environment variables.
     """
     executor_id = environment.get('MESOS_EXECUTOR_ID', 'executor')
     sandbox_directory = environment.get('MESOS_SANDBOX', '')
-    default_progress_output_name = environment.get('EXECUTOR_DEFAULT_PROGRESS_OUTPUT_NAME',
-                                                   '{}.progress'.format(executor_id))
+    default_progress_output_name = \
```

Comment:
Why did this change? I'm not a fan of the trailing slash syntax.

Reply:
I want the `get` arguments to be on the same line. I undid the change by introducing a shorter name variable.